src/configfile.l

```diff
@@ -216,9 +216,18 @@ int evaluatetoken(char *pcToken)
 		}
 		else
 		{
+			static SerialReader *new_reader_list = NULL;
```

Comment:
why a `static` variable?

Reply:
Sorry, it was a copy paste error from the original definition of the reader_list. Removed now.